assets/dev/scss/frontend/_compatibility.scss

```diff
@@ -19,6 +19,7 @@
 	padding: 0;
 }
 
+// To be removed in Elementor 3.26.0.
```

Comment:
We need to remove this code in 3.26.0.

Should we create a ticket for it? 

@doronwolf

Reply:
yes, I believe we should, and give it a proper due date so we won't forget it, I think the 3.26 first beta is in the 25.11
I'll try to verify with Rebbeca